main/src/cgeo/geocaching/connector/lc/LCApi.java

```diff
@@ -62,9 +71,12 @@ protected static Geocache searchByGeocode(final String geocode) {
             return null;
         }
         final Parameters headers = new Parameters(CONSUMER_HEADER, CONSUMER_KEY);
+        final String at = getAlcAccessToken();
+        Log.d("token " + at);
```

Comment:
Is the access token something which should not be shared? If so, it should not be logged in a production version, as log files might be uploaded to github, etc...

Reply:
the access_token is valid for 1 hour .... however, I'll remove the log statement.